.travis.yml

```diff
@@ -38,5 +66,5 @@ script:
   - cat data/sql/updates/db_world/*.sql | mysql -uacore -pacore world
   - mysql -uroot < data/sql/create/drop_mysql.sql
   - cd bin
-  - make -j 8 -k && make install
+  - ninja -j 4 && make install
```

Comment:
what is ninja?

Reply:
https://en.wikipedia.org/wiki/Ninja_(build_system)
I tried this in order to check if it could speed up the build process, but it does not. The build on the main repo takes ca. 40 minutes, if it reaches 50 minutes later on it will be cancelled by the Travis build timeout: https://docs.travis-ci.com/user/customizing-the-build/

Reply:
yeah, i know, we talked about this with rochet, he said we could try to use ccache (which I use on my server) though